### DISCUSSION TITLE
How to create a sidebar menu without child

### DISCUSSION THREAD

**Discussion by ujjwalguptaofficial** [thuliteio/doks#907] [Q&A]
Right now if i create a folder inside docs, it gets added in UI but with collapsible. I just want to make it a link without collpasible as there is no children, so when user clicks on it - it directly show content inside it.

something like - `/docs/get-started`

here is screenshot of the issue - 

![image](https://user-images.githubusercontent.com/10938615/191158628-a65e8196-f2c0-49f5-9569-fff784a1b073.png)

i am not sure how to make this work. Please help me.

**Comment by h-enk** [comment]
Confirmed (bumped into it myself a few days ago). Will be fixed (got no quick, stable + tested fix for now).

**Reply by ujjwalguptaofficial** [reply]
ok please keep me posted. Thanks.

**Reply by ujjwalguptaofficial** [reply]
@h-enk any update on this ?

**Reply by h-enk** [reply]
Add between https://github.com/h-enk/doks/blob/master/layouts/partials/sidebar/auto-collapsible-menu.html#L5-L6:

```html
{{ range .RegularPages }}
  {{ $active := in $currentPage.RelPermalink .RelPermalink -}}
  <li class="mb-1">
    <a class="section-link{{ if $active }} active{{ end }}" href="{{ .Permalink }}">{{ .Title }}</a>
  </li>
{{ end -}}
```

**Reply by inkShio** [reply]
Faced the same problem, your solution is working, thanks. But there is a problem with weight.
